data-median.h

```diff
@@ -0,0 +1,92 @@
+#ifndef __DATA_MEDIAN__
+#define __DATA_MEDIAN__
+
+#define SIZE 9u
+
+int data[SIZE];
+uint8_t current_sample = 0;
+
+void count(double* a, double* inter, int* key, size_t size_a, size_t size_inter)
```

Comment:
Please give more specified name for this methods
Count what?

Reply:
Function counts how much random numbers(0 .. 1) belongs to specified interval.
This data is used later for generation discrete random numbers.

Reply:
that's great - but still give some other name if this is "median-specific" or move to init data if it is "data-specific"